mautic/mautic
# Default preferences center save success message

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ N ]
| New feature/enhancement? (use the a.x branch)      | [ Y ]
| Deprecations?                          | [ N ]
| BC breaks? (use the c.x branch)        | [ N ]
| Automated tests included?              | [ Y ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR adds success message in the default contact preferences page.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Enable "Show contact preference settings" in Mautic configuration
4. Create contact, email and send it to generate unsubscribe link
5. Click unsubscribe link
6. Click "Save" in preferences page
7. Success message should appear on the page
![obraz](https://user-images.githubusercontent.com/8580942/189833069-7c23749f-f441-4b10-930a-e69a130d2a52.png)
